src/tests/unit/tests/common/components/cards/__snapshots__/rule-resources.test.tsx.snap

```diff
@@ -190,10 +197,13 @@ exports[`RuleResources renders with {
 exports[`RuleResources renders with {
   url: 'test-url',
   guidanceLinks: [ { href: 'test-href' }, [length]: 1 ],
-  linkComponent: 'ExternalLink'
+  linkComponent: 'ExternalLink',
+  outcomeType: 'pass'
 }: GuidanceTags props 1`] = `
 {
   "deps": {
+    "GetNeedsReviewRuleResourcesUrl": [Function],
```

Comment:
Hmm....

We don't have direct unit tests for `needs-review-rule-resources`, and are instead testing that code via `rule-resources-test.tsx`. The preferred pattern is for 1 test class to exercise 1 production class, and to mock out dependencies. If we were following our usual pattern, both `GetNeedsReviewRuleResourcesUrl` and `IsOutcomeNeedsReview` would be `MockFunction` instead of `Function`, and there would e a `needs-review-rule-resources.test.tsx`

@madalynrose, I'd love to hear your thoughts on this...

Reply:
changed the `GetNeedsReviewRuleResourcesUrl` and `IsOutcomeNeedsReview` to be `MockFunction`. we have a seperate test file `needs-review-rule-resources.test.tsx` for the above functions.